Swap sign of flux divergence

btobers opened the pull request:
Closes https://github.com/OGGM/oggm/issues/1814

This PR addresses an issue with the sign of the flux divergence stored within the flowline diagnostics.

- [x] Tests added/passed
- [ ] Fully documented
- [ ] Entry in `whats-new.rst`

btobers committed:
Swap sign of flux divergence

btobers committed:
Added to whats-new.rst

fmaussion commented:
@pat-schmitt are you fine with this? We will have to adapt the tests and the documentation / tutorials as well.

pat-schmitt commented on oggm/core/flowline.py:
```diff
@@ -1347,7 +1347,7 @@ def run_until_and_store(self, y1,
                             surface_h_previous[fl_id] = fl.surface_h
                         if 'flux_divergence' in ovars_fl and (yr > self.y0):
                             # calculated after the formula dhdt = mb + flux_div
```
change this to '# calculated after the formula mb = dhdt + flux_div'

pat-schmitt commented:
Thanks @btobers for spotting this. I used the wrong equation. I just added a comment to also change the comment above the new calculation. I can have a look at the tests the next weeks.

pat-schmitt committed:
Update comment above flux divergence calculation

pat-schmitt commented:
I just changed the comment to run the tests again

pat-schmitt committed:
Refactor flux divergence calculation in test

pat-schmitt committed:
trigger test workflow

pat-schmitt commented:
Beside of changing documentation and tutorials this is ready

fmaussion committed:
Merge branch 'master' into 1814_flux_div_sign

fmaussion committed:
Fix entry from whats-new.rst

Removed duplicate entry for flowline diagnostics flux divergence fix in the Enhancements section.

fmaussion committed:
add test
